<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactManagerDlg</class>
 <widget class="QDialog" name="ContactManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Contacts Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ContactManagerView" name="usersView">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblField">
       <property name="text">
        <string>Field</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbField"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMatchType"/>
     </item>
     <item>
      <widget class="QLineEdit" name="edtMatch">
       <property name="maximumSize">
        <size>
         <width>32000</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">+/-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblAction">
       <property name="text">
        <string>Action</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbAction"/>
     </item>
     <item>
      <widget class="QLineEdit" name="edtActionParam">
       <property name="toolTip">
        <string extracomment="Action's parameter"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbActionParam">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExecute">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactManagerView</class>
   <extends>QTableView</extends>
   <header>contactmanager/contactmanagerview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContactManagerDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContactManagerDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtMatch</sender>
   <signal>returnPressed()</signal>
   <receiver>btnSelect</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
